spel/minimal-linux.pkr.hcl

```diff
@@ -539,7 +539,15 @@ variable "amigen8_source_url" {
 variable "amigen8_storage_layout" {
   description = "List of colon-separated tuples (mount:name:size) that describe the desired partitions for LVM-partitioned disks on EL8 images"
   type        = list(string)
-  default     = []
+  default = [
+    "/:rootVol:5",
```

Comment:
Only increasing the root LVM volume by 1GiB? Is that going to be sufficient beyond the near-term?